java/java.lsp.server/src/org/netbeans/modules/java/lsp/server/db/DBConnectionsProvider.java

```diff
@@ -26,51 +26,51 @@
 import java.util.Map;
 import java.util.Set;
 import java.util.concurrent.CompletableFuture;
-import java.util.stream.Collectors;
 import org.eclipse.lsp4j.CodeAction;
 import org.eclipse.lsp4j.CodeActionParams;
 import org.netbeans.api.db.explorer.ConnectionManager;
 import org.netbeans.api.db.explorer.DatabaseConnection;
 import org.netbeans.modules.java.lsp.server.protocol.CodeActionsProvider;
 import org.netbeans.modules.java.lsp.server.protocol.NbCodeLanguageClient;
-import org.netbeans.modules.java.lsp.server.protocol.UpdateConfigParams;
 import org.netbeans.modules.parsing.api.ResultIterator;
 import org.openide.util.lookup.ServiceProvider;
 
 /**
- * Updates Run Configuratoin with Database env variables.
  * 
  * @author Jan Horvath
  */
 @ServiceProvider(service = CodeActionsProvider.class)
-public class DBSetEnvCommand extends CodeActionsProvider {
-    private static final String  COMMAND_SET_DB_ENV = "java.db.set.env"; //NOI18N
-    private static final String CONFIG_SECTION = "java+.runConfig"; //NOI18N
+public class DBConnectionsProvider extends CodeActionsProvider{
+    private static final String  GET_DB_CONNECTIONS = "java.db.connections"; //NOI18N
```

Comment:
Why the class and command is named 'connections', when it returns information about one DB connection?

Reply:
Originally it was meant to provide all db connection in workspace. Fixed